rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -94,6 +94,9 @@ find /srv/salt -maxdepth 1 -type f -iname '*.top' \
     | xargs qubesctl top.enable > /dev/null
 
 %changelog
+* Fri Feb 14 2020 Kevin O Gorman <kog@freedom.press> - 0.1.4
```

Comment:
Per @conorsch the recommendation for future changelog entries is to use "SecureDrop Team <securedrop@freedom.press>" consistently

Reply:
Citation: https://github.com/freedomofpress/securedrop-debian-packaging/issues/108

Reply:
:heavy_check_mark: